### DISCUSSION TITLE
Trips location not detailled

### DISCUSSION THREAD

**Discussion by valerossibzh** [flobz/psa_car_controller#968] [Q&A]
Hello!
I recently integate your controller to my homeassistant with success.
First, thanks for your work, that's very usefull!
But when i check the map data, i only see some straight lines between start and stop point. It seem's that API doesn't send te detailled location during moving. I saw that other persons have the real trace of the vehicule.
Can i configure something to have the details of trips?
Do you have these data?
Thanks!

**Comment by Jordan87** [comment]
In my case a little more detailed trips appeared after activation [MY TRIP REPORT](https://services-store.peugeot.co.uk/store/my-trip-report) which enable cloud sync for trips, before activation it was only start and endpoint.

**Comment by valerossibzh** [comment]
Thanks. I tried to register but I have an error that my car (e208) is not compatible...
I saw somewhere else that someone had also more data after using eevee app, but it's a paid app 😞

**Reply by Jordan87** [reply]
I was seeing similar info on the beggining but finaly I was able to activate My trip report for my 308 Plug-in hybrid.
try from app or after logging to https://services-store.peugeot.co.uk/login-redirect ther should be a list of available services.

**Comment by valerossibzh** [comment]
I checked, i can only subscribe to "e-routes" service as paid option...
May be by using app  "A better route planner"? I will give a try.

**Comment by sod75** [comment]
For me at first I also had the straight lines, then after subscribing to the eevee app it started working but has now stopped a while back and the app indeed is paying now.
I guess this means that they stopped this server side because they started charging for it and eevee is not willing to foot the bill for free accounts.
If anyone gets it running(without a payed subscription) I'd be happy to hear how though...

**Comment by valerossibzh** [comment]
I downloaded and configured the Android app "Tronity". There is a 14-day free trial period. 24 hours later, my car trips started to be detailed, I can see the real trace in the PSA controller.
Since I didn't give any payment information, I don't know if it will work after the 14 days, probably not!
I will let you know...

**Reply by valerossibzh** [reply]
For information, the data received in PSA Car Controller is actually linked to other applications. During the 14-day trial of the Tronity app, my PSA car controller displayed detailed journeys on the map. As soon as my vehicle was disconnected from Tronity, it stopped working and I only had the starting and ending point of the trips.
I think business apps pay a fee to Stellantis to get detailed data.
I haven't found any free apps that provide Stellantis data, please let me know if anyone finds a solution to enable this data.

**Comment by nickl1984** [comment]
What worked for me:

1.  Try to activate [My trip report](https://connect.opel.nl/store/my-trip-report)
2. On the site it wasn't available for my car so no luck there but after allowing the 'My Opel app' bluetooth connection to the car and driving for a while the service appeared on the app (On the services tab). Activated it and now the app shows the routes, not detailed but just time, begin and end point. Even if i don't connect my phone to the car now i still receie the data.
3. After this i started the PSACC addon in homeassistant with the option -r and -c enabled (-r -c --web-conf). (The -c option was critical in this because without it i never received any updates from the car, only after a manual refresh)
4. Now the app shows all the data coming from the car included detaild trip reports (whole route, not just begin and end point) and SOC etc. refreshes correctly
5. While charging i often get the Rate Limit Exeption but it doensn't seem to affect anything, data is just coming in normally.

Good luck ;)
